scripts/minikube-k8sibmc-all-tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
Can you add some comment after this line to describe the purpose of this shell script? Also, does it need `minikube` in its name? I'm asking because my understanding is that we're moving to IBM k8s cluster and are not going to use minikube any more.

Reply:
renaming it to remove minikube